geonode_mapstore_client/client/js/components/FiltersForm/FilterByExtent.jsx

```diff
@@ -123,22 +137,20 @@ function FilterByExtent({
                                 type: 'vector',
                                 features: [getFeatureFromExtent(queryExtent)],
                                 style: vectorLayerStyle
-                                    ? vectorLayerStyle
+                                    ? {...vectorLayerStyle,  weight: 0.001}
```

Comment:
change to trigger the pull request check again
```suggestion
                                    ? { ...vectorLayerStyle,  weight: 0.001 }
```